webapps-backup/src/main/java/io/camunda/webapps/backup/repository/opensearch/SnapshotState.java

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+package io.camunda.webapps.backup.repository.opensearch;
+
+public enum SnapshotState {
```

Comment:
Copied from operate, we can remove the one in operate when we remove the `OpenSearchSnapshotOperations` class